src/json-completion.ts

```diff
@@ -66,6 +83,7 @@ export class JSONCompletion {
     }
 
     const currentWord = getWord(ctx.state.doc, node);
+    const rawWord = getWord(ctx.state.doc, node, false);
```

Comment:
this logic is only needed for `json5` really, so we can guess how to insert the completion text based on quotation types or lack thereof